modules/router/templates/googlec908b3bc32386239.html.erb

```diff
@@ -0,0 +1 @@
+google-site-verification: googlec908b3bc32386239.html
```

Comment:
This is in templates, but it's not a template, as the content is fixed. It could be moved to the files directory, but also as there's a single line, the contents could be moved inside the NGinx configuration, I'll comment more above.